### ISSUE TITLE
Delete /script/ci files when all Buildkite pipelines moved to ofn-buildkite

### ISSUE THREAD

**Issue by pmackay** [openfoodfoundation/openfoodnetwork#1422]
It would be good to avoid having the two copies of the CI scripts. Some pipelines need converting over:

- [ ] Convert `Open Food Network Deploy`
- [ ] Convert `Open Food Network Staging1`

The other 2 pipelines appear to be converted already. @mkllnk could you please confirm this and say what is needed to convert? 

Also should `ofn-install` support installing `ofn-buildkite`?

**Comment by mkllnk** [comment]
> Convert Open Food Network Deploy

I guess we would like to get rid of that one anyway. I have to confirm that with the team.

> Convert Open Food Network Staging1

Yes. We need to do that.

> The other 2 pipelines appear to be converted already.

That are Open Food Network Pull Requests and UK Deployment. UK Deployment actually still uses `script/ci/check_github_status.sh`.

> Also should ofn-install support installing ofn-buildkite?

That would be nice to have, but we set it up only once. I don't know if there is a need for it.

**Comment by pmackay** [comment]
>That would be nice to have, but we set it up only once. I don't know if there is a need for it.

Isnt it needed on any server that receives deploys from Buildkite though?

**Comment by mkllnk** [comment]
There are two parts of the scripts in `script/ci`. The files that are in ofn-buildkite are only for the Buildkite agent, that's only one server at the moment. But there are also two scripts that are executed on the OFN servers:

- `save_staging_baseline.sh`
- `load_staging_baseline.sh`

I'm not sure about the best way to handle these. The commands in `save_staging_baseline.sh` could easily be called remotely by the ofn-buildkite scripts. And `load_staging_baseline` could be part of the `post-receive` Git hook. I would probably still keep them in the `script` directory of the OFN code.

**Comment by lin-d-hop** [comment]
I think that this is now irrelevant with ofn-install.
Reopen if you disagree....
